src/utils/AddressUtils.ts

```diff
@@ -68,3 +70,170 @@ export function isValidEvmAddress(address: string): boolean {
     return false;
   }
 }
+
+// Creates the proper address type given the input chain ID corresponding to the address's origin network.
+// @todo: Change this to `toAddress` once we remove the other `toAddress` function.
```

Comment:
NIT: We should try to use TSDoc style comments for this type of function/method documentation.

Reply:
Good call. https://github.com/across-protocol/sdk/pull/909/commits/92539941d0dcc6c8a974822b9c84d5c5e524a465.

I think it is generally good to have a single comment format. Should all functions, including class methods, be TSDoc style as well?